test/unit/lib/utils.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+const utils = require('../../../app/lib/utils');
+
+const expect = chai.expect;
+
+describe('utils', () => {
+  describe('deepClone', () => {
```

Comment:
Since `JSON.parse(JSON.stringify(obj));` is such a common implementation of a 'poor mans clone' isn't this just testing javascript implementation.